public/mockServiceWorker.js

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable */
```

Comment:
this is automatically generated by MSW

Reply:
It doesnt have to be called "public", we just need something that is meant to be static so that storybook can access it to publish to and access the mockserviceworker. What would your suggestion be?

Reply:
moved it under `./storybook`